spec/features/user_votes_for_a_review_spec.rb

```diff
@@ -62,4 +63,26 @@
     expect(page).to have_content "Score: 1"
   end
 
+    scenario "a user successfully votes and gets email" do
+    sign_in(@user)
+
+    ActionMailer::Base.deliveries =[]
+
+    visit country_city_path(@country, @city)
+
+    find("#up_vote").click
+
+    expect(page).to have_content "You have voted!"
+    expect(page).to have_content "Score: 1"
+
+
+    expect(ActionMailer::Base.deliveries.size).to eql(1)
+
+    last_email = ActionMailer::Base.deliveries.last
+    expect(last_email).to have_subject("Post Upvoted")
+    expect(last_email).to deliver_to(@creator.email)
+  end
```

Comment:
`end` at 84, 2 is not aligned with `scenario "a user successfully votes and gets email" do` at 66, 4